docs/monitoring.md

```diff
@@ -1291,6 +1291,17 @@ These metrics are exposed by Spark executors.
   - shuffleRemoteBytesReadToDisk.count
   - shuffleTotalBytesRead.count
   - shuffleWriteTime.count
+  - Metrics related to push-based shuffle:
```

Comment:
Are these always exposed when we don't use `push-based shuffle`?

Reply:
Yes, they are. I understand https://issues.apache.org/jira/browse/SPARK-42203 is supposed to take care of that issue.

Reply:
Thanks! I pinged on SPARK-42203. Let's wait for a few days.